superset/connectors/base/models.py

```diff
@@ -152,6 +152,12 @@ def short_data(self):
             'creator': str(self.created_by),
         }
 
+    @property
```

Comment:
This method belongs in `connectors.sqla.models.SqlaTable` as Druid datasources don't linke to `Database` but to cluster, this base should have a `raise NotImplementedError()` or `pass`.

Also in that context you'll be able to simply use `self.database.select_star(...)`